### ISSUE TITLE
Wrong version ranges

### ISSUE THREAD

**Issue by knqyf263** [bitnami/vulndb#133]
### Title

CVE-2020-12603

### What steps will reproduce the bug?

The Bitnami advisory has only `introduced` versions here.
https://github.com/bitnami/vulndb/blob/c75104a0d9649e32a458c69cbb71f2c782696ce8/data/envoy/BIT-2020-12603.json#L33-L48

However, the vulnerability should be fixed by 1.14.3 and 1.13.3.
[GHSA-pc38-4q6c-85p6](https://github.com/envoyproxy/envoy/security/advisories/GHSA-pc38-4q6c-85p6)
https://groups.google.com/g/envoy-announce/c/qrrF8klFl-I/m/nz12XtqmAAAJ?pli=1

NVD pins the exact version.
https://nvd.nist.gov/vuln/detail/CVE-2020-12603

>  cpe:2.3:a:envoyproxy:envoy:1.13.2:*:*:*:*:*:*:*
>  cpe:2.3:a:envoyproxy:envoy:1.14.2:*:*:*:*:*:*:*

There are two problems.

1. Fixed versions in NVD are wrong in the first place
2. Bitnami wrongly parses NVD

### What is the expected behavior?

Ideally, it should have the fixed version.

```
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "1.13.2"
            }
            {
              "fixed": "1.13.3"
            }
          ]
        },
```

At least, it should pin the affected version.

```
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "1.13.2"
            }
            {
              "last_affected": "1.13.2"
            }
          ]
        },
```

or

```
versions:
  - 1.13.2
```

I gave an example of 1.13.x, but 1.14.x is the same.

### What do you see instead?

```
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "1.13.2"
            }
          ]
        },
```

**Comment by juan131** [comment]
Thanks so much for reporting this issue @knqyf263!! I'll sync with @gongomgra to properly investigate and address it.

**Comment by gongomgra** [comment]
Hi @knqyf263,

Thank you so much for reporting this issue. I have created an internal task to address it.

**Comment by gongomgra** [comment]
Hi @knqyf263,

Thanks for your update. We have updated the database as per the conversation in [trivy-db/#352](https://github.com/aquasecurity/trivy-db/pull/352/). Let us know if there is anything else pending on our side.

**Comment by knqyf263** [comment]
I confirmed other vulnerabilities were also fixed! Thanks!
https://github.com/bitnami/vulndb/blob/501081da764e8c70267696bff7e369693233e66f/data/postgresql/BIT-2021-23214.json#L6

**Comment by knqyf263** [comment]
Just FYI: I noticed this was simpler.

```
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0.1.0"
            },
            {
              "fixed": "1.0.0"
            }
            {
              "introduced": "1.1.0"
            },
            {
              "last_affected": "1.1.0"
            }
          ]
        }        
      ]
```

rather than

```
      "ranges": [
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "0.1.0"
            },
            {
              "fixed": "1.0.0"
            }
          ]
        },
        {
          "type": "SEMVER",
          "events": [
            {
              "introduced": "1.1.0"
            },
            {
              "last_affected": "1.1.0"
            }
          ]
        }        
      ]
```
